### PR TITLE
Feature | Record Requests In Telescope

### LINKED ISSUE #21: Send requests / responses to laravel telescope?
Hi, first thanks for the nice package, but as always it's still room to improve it.

Laravel telescope has a really nice feature **[HTTP Client](https://laravel.com/docs/9.x/telescope#http-client-watcher)** watcher records outgoing [HTTP client requests](https://laravel.com/docs/9.x/http-client) made by your application. 

telescope

Is it plan to send requests / responses to laravel telescope?

### PR DESCRIPTION
**Pull request by Sammyjo20**
This PR introduces a new listener which can be subscribed to so that it records requests in Laravel Telescope. The telescope entry will be recorded under the "HTTP Client" section of Telescope. 

Fixes #21

### CODE REVIEW

**Review by Sammyjo20 — COMMENTED** [review]
Self review complete

File: phpunit.xml [hunk 1]
```diff
@@ -18,5 +18,7 @@
     <php>
         <env name="APP_KEY" value="base64:g6rX3ygsFZiO446CYsmZVbPZBGavoH0sVbGXZbS03SI="/>
         <env name="BCRYPT_ROUNDS" value="4"/>
+         <env name="DB_CONNECTION" value="sqlite"/>
```
Comment (rrolla) [inline phpunit.xml]: Need small formatting fix for both envs

**Review by rrolla — APPROVED** [review]
Fantastic :)

**Comment by Sammyjo20** [comment]
@rrolla Any idea why these tests are failing? I'm struggling to find out how to fix it :/

**Comment by rrolla** [comment]
> @rrolla Any idea why these tests are failing? I'm struggling to find out how to fix it :/ 

@Sammyjo20 Can you please provide more information? What's the error, how to run tests?

**Comment by rrolla** [comment]
Oh from mobile I cannot see here is github actions, I will take a look.

**Comment by rrolla** [comment]
@Sammyjo20 I would say its because of this package updates: https://github.com/ramsey/uuid/releases but I don't know which package depends on ramsey/uuid

**Comment by Sammyjo20** [comment]
Closing as this will be a v2 feature natively 🚀

**Comment by rrolla** [comment]
Thanks @Sammyjo20 waiting for the release :)

**Comment by Sammyjo20** [comment]
Keep an eye out on the [issue](https://github.com/Sammyjo20/Saloon/issues/64) for updates :)

**Pull request closed without merging**